common/src/main/resources/assets/figura/lang/en_us.json

```
@@ -692,12 +693,12 @@
     "figura.docs.animation.play": "Initializes the animation\nResume the animation if it was paused",
     "figura.docs.animation.pause": "Pause the animation's playback",
     "figura.docs.animation.stop": "Stop the animation",
-    "figura.docs.animation.restart": "Restarts the animation\nPlays the animation if it was stopped\nThis behaviour can also be reproduced by stopping then playing the animation",
+    "figura.docs.animation.restart": "Restarts the animation\nPlays the animation if it was stopped\nThis behavior can also be reproduced by stopping then playing the animation",
     "figura.docs.animation.set_playing": "A function that merges \"play\" and \"stop\" together\nTakes a boolean parameter, where if true, the animation will play, and when false, the animation will stop",
     "figura.docs.animation.get_time": "Get the animation's playback current time",
     "figura.docs.animation.set_time": "Sets the animation's playback current time",
     "figura.docs.animation.get_play_state": "Get the animation's playback state",
-    "figura.docs.animation.new_code": "Adds a string to run in a determinant time\nIf theres already code to run at that time, it is overridden",
+    "figura.docs.animation.new_code": "Adds a string to run in a determinant time\nIf there's already code to run at that time, it is overridden",
```

Maybe "overwritten" would fit better here since the code that was originally at that time is deleted.
```suggestion
    "figura.docs.animation.new_code": "Adds a string to run in a determinant time\nIf there's already code to run at that time, it is overwritten",
```